src/components/governance/EditStream.tsx

```diff
@@ -1399,7 +1400,13 @@ export default function EditStream(props: EditStreamProps) {
                 target="_blank"
                 href={`https://twitter.com/intent/tweet?text=I%20just%20opened%20a%20contribution%20stream%20to%20${
                   isFundingMatchingPool ? "the SQF Matching Pool" : granteeName
```

Comment:
@tnrdd Minor thing here. We end up with the Grantee Name & then their Twitter handle in the text.